frontend/src/metabase/dashboard/components/DashboardInfoSidebar/DashboardInfoSidebar.styled.tsx

```diff
@@ -5,9 +5,10 @@ import { breakpointMaxSmall } from "metabase/styled-components/theme";
 import { color } from "metabase/lib/colors";
 import EditableText from "metabase/core/components/EditableText";
 import FormField from "metabase/core/components/FormField/FormField";
+import { SIDEBAR_WIDTH } from "../Sidebar";
 
 export const DashboardInfoSidebarRoot = styled.aside`
-  width: 360px;
+  min-width: ${SIDEBAR_WIDTH}px;
```

Comment:
Make dashboard sidebar widths consistent. The info sidebar had a different width from adding question, subscriptions, and filters sidebar. This plays into adjusting the dashboard header positioning to align with dashboard body positioning when sidebar is open (https://github.com/metabase/metabase/pull/38634/files#diff-382948855bb5f3cf36bc9f62e5b266b4d7943473b4b30b4fd26890062f5eddbbR120)